.github/workflows/e2e_nightly_upgrade.yml

```diff
@@ -53,13 +53,27 @@ jobs:
 
         ps_target_version: ${{ fromJson(needs.get_ps_target_versions.outputs.target_versions) }}
 
+        module_branch: ['dev', 'master']
+
       fail-fast: false
 
     steps:
 
       # Setup PrestaShop and Install Autoupgrade module
       - uses: actions/checkout@v2
 
+      - run: |
+          cp -r .github .github_dev
+          cp -r tests/e2e tests/e2e_dev
+          cp docker-compose.yml docker-compose.dev.yml
+          git fetch origin ${{ matrix.module_branch }}
+          git checkout ${{ matrix.module_branch }}
+          rm -rf .github docker-compose.yml tests/e2e
+          mv .github_dev .github
+          mv tests/e2e_dev tests/e2e
+          mv docker-compose.dev.yml docker-compose.yml
+        if: ${{ matrix.module_branch != 'dev' }}
```

Comment:
Since some directories files are not merged on the master branch, adding the same files from the dev branch to be able to run tests.